src/shared/dropdown-wrapper/dropdown-wrapper.styles.tsx

```diff
@@ -18,49 +18,67 @@ export interface DropdownWrapperStyleProps {
 
 export interface ValueLabelStyleProps {
     truncateType?: TruncateType;
+    variant?: DropdownVariantType | undefined;
+}
+
+export interface SelectorStyleProps {
+    variant?: DropdownVariantType;
 }
 
 // =============================================================================
 // STYLING
 // =============================================================================
 const BORDER_RADIUS = "4px";
+const _getHeight = (variant?: DropdownVariantType | undefined) => {
+    // 2.5rem 40px, 3rem 48px
+    return variant === "small" ? 2.5 : 3;
+};
 
-export const Wrapper = styled.div`
+export const Wrapper = styled.div<SelectorStyleProps>`
     position: relative;
-    min-height: 3rem;
-    height: 3rem; // Need this to persist the height when expanding or collapsing list
     width: 100%;
+    ${(props) => {
+        const height = _getHeight(props.variant);
+        return css`
+            min-height: ${height}rem;
+            height: ${height}rem; // Need this to persist the height when expanding or collapsing list
+        `;
+    }}
 `;
 
-export const baseSelectorCSS = css`
-    display: flex;
-    position: relative;
-    align-items: center;
-    justify-content: space-between;
-    padding: 0 1rem;
-    height: calc(3rem - 2px); // exclude top and bottom borders
-    width: 100%;
-    border-radius: ${BORDER_RADIUS};
-    border: none;
-    background: transparent;
-
-    :focus,
-    :active {
-        outline: none;
-    }
+export const baseSelectorCSS = (props?: SelectorStyleProps) => {
+    const height = _getHeight(props.variant);
```

Comment:
instead of converting to a function, `css` can also take in props

```
export const baseSelectorCSS = css<SelectorStyleProps>`
  ${props => ...}
`
```